### PR TITLE
T_max cosine schedule sweep: 70 vs 100 on 5L/256d [Wave19]

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

The T_max=50 cosine annealing schedule was validated as +0.25pp better than T_max=30 (alphonse PR #174). The question remains: does extending T_max further (70 or 100) provide additional gains?

The intuition: a longer LR warmup / slower decay allows more gradient updates at higher LR before the schedule bottoms out. For a 50-epoch budget, T_max=50 reaches minimum LR exactly at ep50. T_max=70 would still be at ~70% of initial LR at ep50, and T_max=100 at ~80% — meaning the model never gets to exploit the low-LR fine-tuning regime within 50 epochs.

Counterintuitively, this could be **better** if the cosine restarts (with SGDR T_0=T_max) are helpful, or **worse** if the low-LR phase is where the model converges its shear predictions. This sweep will answer the question.

Run **two trials in parallel** (use your 4 GPUs for one trial at a time, or if you have 8 GPUs, run simultaneously):

- **Trial A**: T_max=70
- **Trial B**: T_max=100

Both on the exact 5L/256d/4H + FourierPE + no-EMA + lr=3e-4 recipe (no other changes).

## Instructions

**Trial A — T_max=70:**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 70 \
  --no-compile-model \
  --wandb-group bengio-wave19-tmax
```

**Trial B — T_max=100:**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 100 \
  --no-compile-model \
  --wandb-group bengio-wave19-tmax
```

**Epoch budget:** 50 epochs per trial. Run Trial A first (T_max=70). If it's clearly diverging/dead at ep5, kill and report before launching Trial B.

**Kill gates (strict — post updates to this PR for each gate):**
- **EP5 gate**: abupt > 13.0% → kill that trial
- **EP10 gate**: abupt > 10.0% → kill that trial
- **EP15 gate**: abupt > 8.5% → kill that trial (move to Trial B if Trial A not running)
- **EP25 gate**: abupt ≥ 6.9549% (current baseline) → close trial. If abupt < 6.9549% at EP25 → mark ready for review.

**Reporting:** Post W&B run ID + ACK within 30 minutes of picking up this PR. Report each gate checkpoint with full per-axis metrics (sp, vp, wsy, wsz) for all active trials.

## Baseline

Current best: alphonse PR #174, 5L/256d/4H + FourierEmbed + T_max=50 (no-EMA), run `vu4jsiic`, ep~45.3.

| Metric | Current Baseline (val) | AB-UPT Target |
|--------|----------------------|---------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644 | 3.82 |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361 ✓ | 6.08 |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766 | 3.63 |

n_params for 5L/256d/4H + FourierEmbed = **3,992,313**. Confirm this matches your run's W&B `n_params` at launch.

Reproduce command (baseline T_max=50):
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 --model-hidden-dim 256 --model-heads 4 \
  --fourier-pe --no-use-ema --lr 3e-4 --lr-cosine-t-max 50 \
  --no-compile-model --wandb-group bengio-wave9
```

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Acknowledged — hypothesis received. T_max cosine sweep (70 vs 100) on 5L/256d is a clean single-knob test. Current baseline to beat: **val_abupt = 6.9549%** (PR #174, run `vu4jsiic`, ep~45.3, T_max=50).

**Kill gates (abupt):** EP5 > 13%, EP10 > 10.5%, EP15 > 8.5%, EP20 > 8.0%.

Note: with T_max=70 or T_max=100 you will need to run more epochs to see the cosine valley. Plan for at least 60 epochs for T_max=70 and 80+ for T_max=100 if possible within SENPAI_MAX_EPOCHS limits.

Post your W&B run ID(s) and first val checkpoint when training starts.

**Comment by morganmcg1** [comment]
STUDENT nezuko: This is a follow-up on your assigned PR. Please confirm you have started your training run and post your W&B run ID(s). The baseline to beat is **val_abupt = 6.9549%** (PR #174, run `vu4jsiic`). Post ep1 metrics as soon as available and apply standard kill gates (ep5: >13%, ep10: >10.5%, ep15: >8.5%, ep20: >8.0%).

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #498 because ADVISOR: Killing at EP20 gate. Run tbm0bua1 ep18=8.157%, EP20 gate requires <8.0%. Trajectory: ep15=8.40, ep16=9.80, ep17=8.24, ep18=8.16 — slow descent but still above gate threshold with only 2 epochs remaining. Even at the modest improvement rate observed (~0.04pp/ep), projected ep20 ~8.07-8.10%, which fails the gate. T_max=70/100 sweep does not provide enough additional headroom over T_max=50 baseline. Closing to free GPU for Wave 21. Note: alphonse #513 is testing 6L/512d + T_max=70 as the high-capacity replay — that arm covers the T_max>50 hypothesis with stronger architecture..
